assets/src/edit-story/components/form/toggle.js

```diff
@@ -89,6 +90,7 @@ function Toggle({
           onChange={() => onChange(!value)}
           disabled={disabled}
           title={title}
+          aria-label={ariaLabel}
```

Comment:
These last three properties are just copied straight from the props. Could we rewrite to just `...rest`? Then we can remove them from validation, but then also re-write `ariaLabel` to `aria-label` where used? And maybe do that for all components?

I know that means, that we don't validate that components have labels then, but we also avoid all this senseless prop drilling.

Reply:
Part of the point of this PR was to bring the label out of the `...rest` so that it would be specifically required to ensure labels are actually used in the future. In various places, the label was part of `...rest` and this resulted in not specifying one. Any other suggestions on how to enforce using labels?

Perhaps we could move some of the other properties to `...rest` which are not required, to clean it up more, such as `disabled`. Thoughts?

Reply:
Well, I suppose the reviews and QA should catch the missing labels. I was just hoping to visibly make it required! Will refactor the PR, thanks for the informative comments.